content/bluetooth.md

```diff
@@ -70,6 +71,19 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 ## Bluetooth Troubleshooting
 
+<!--## Dear Pop OS team I hope this helps to improve your distribution and documentation, feel free to rework.-->
+### Problem: Fresh installed pop-os_22.04_amd64_intel_38.iso, Bluetooth dont turn on
```

Comment:
This doesn't need to be a whole subsection right at the beginning of the Troubleshooting section.